core-api/README.md

```diff
@@ -1,6 +1,8 @@
 # Java SDK Core API
 This package contains the core APIs and interfaces for the Optimizely Full Stack API in Java.
 
+The full product documentation can be found on the the Optimizely developers site [here](https://docs.developers.optimizely.com/full-stack/docs/welcome).
+
```

Comment:
Suggest:
Full product documentation is in the the [Optimizely developer documentation](https://docs.developers.optimizely.com/full-stack/docs/welcome).